virtio-devices/src/block.rs

```diff
@@ -543,11 +543,9 @@ impl<T: 'static + DiskFile + Send> VirtioDevice for Block<T> {
             thread::Builder::new()
                 .name("virtio_blk".to_string())
                 .spawn(move || {
-                    if let Err(e) = SeccompFilter::apply(virtio_blk_seccomp_filter) {
-                        error!("Error applying seccomp filter: {:?}", e);
-                    } else if let Err(e) = handler.run(paused) {
-                        error!("Error running worker: {:?}", e);
-                    }
+                    SeccompFilter::apply(virtio_blk_seccomp_filter)
+                        .map_err(EpollHelperError::ApplySeccompFilter)?;
```

Comment:
I made Bo remove this from `EpollHelperError` as it is not an error generated by `EpollHelper`. Also you really want to print the message out here as we never join in the epoll_threads and so the messages get lost. See #1551.

Reply:
Ok got it!